### PR TITLE
Add (... unstable) version to a few library names

### PR DESCRIPTION
**Pull request by lassik**
Also update Chicken egg.

### CODE REVIEW

**Comment by lassik** [comment]
`scripts/reinstall-chicken-5-egg.sh` will build and install the egg.

File: live.egg [hunk 1]
```diff
@@ -7,13 +7,51 @@
...
+   (extension
+     live.fixnum.unstable
+     (source "live/fixnum/unstable.sld")
+     (source-dependencies)
+     (component-dependencies)
+     (csc-options "-R" "r7rs" "-X" "r7rs"))
+   (extension
+     live.list.unstable
+     (source "live/list/unstable.sld")
+     (source-dependencies)
```
Comment (rgherdt) [inline live.egg:37]: we should at some point add the .scm files as source-dependencies

**Review by rgherdt — APPROVED** [review]

File: live.egg (hunk 1 above)
Comment (lassik) [reply live.egg:37]: Good catch. That will require a groveler that reads each `.sld` file and finds the includes.

Comment (lassik) [reply live.egg:37]: I'll make a separate PR that adds the groveler.

**Pull request merged**